src/data/SpreedSignaling.ts

```diff
@@ -0,0 +1,170 @@
+// NOTE(andrews): This file contains most of the messages that
```

Comment:
What's the plan for supporting other webRTC service?

Should we have a more general interface name?